CONTRIBUTING.adoc

```diff
@@ -0,0 +1,90 @@
+= Contributing to Kubernetes the AWSome Way
+Thank you for your interest in contributing to Kubernetes the AWSome Way! We work hard to provide a high quality and useful workshop for our customers, and we appreciate your interest in helping us and the rest of our community of users. We welcome bug reports, enhancements, and contributions.
+
+__Jump To:__
+
+* link:#bug-reports[Bug Reports]
+* link:#enhancements[Enhancements]
+* link:#contributions[Contributions]
+* link:#getting-in-contact[Getting in Contact]
+
+== Bug Reports
+Bug reports are accepted through the link:https://github.com/aws-samples/aws-workshop-for-kubernetes/issues[Issues] page.
+
+The following labels are used to track bug related issues: link:https://github.com/aws-samples/aws-workshop-for-kubernetes/labels/Bug[Bug].
+
+=== Before Submitting a Bug Report
+Before submitting a bug report, please do the following:
+
+1. Do a search through the existing issues to make sure it has not already been reported. If there's an existing one, be sure give a +1 reaction which will help us prioritize which issues to address first.
+
+2. Provide as much information about your environment, software version, and relevant dependencies as possible. For example, let us know what version of kubernetes you are using, which launch type for the kubernetes cluster (e.g. EKS or kops), and the environment your code is running in. (e.g Cloud9).
```

Comment:
Can we create a template for the basic needed information?

Reply:
+1 on Template